packages/components/src/focal-point-picker/index.js

```diff
@@ -223,10 +221,10 @@ export class FocalPointPicker extends Component {
 			( top - bounds.top ) / ( pickerDimensions.height - bounds.top * 2 );
 
 		// Enables holding shift to jump values by 10%
-		const step = byTenths ? 0.1 : 0.01;
-
-		nextX = roundClamp( nextX, 0, 1, step );
-		nextY = roundClamp( nextY, 0, 1, step );
+		if ( byTenths ) {
+			nextX = Math.round( nextX / 0.1 ) * 0.1;
+			nextY = Math.round( nextY / 0.1 ) * 0.1;
+		}
```

Comment:
The values are clamped a few lines before this making the clamp of `roundClamp` redundant.

The larger story is one of yak-shaving. This PR got started when working on another PR (#34566) and wanting to modify `roundClamp` for a more specific purpose. I noticed the only other place it was used was this component, spotted the partial redundancy and figured it could be removed here to potentially free it up for the repurposing.

Reply:
Makes sense. I'm going to keep this conversation open for extra visibility in case anyone else reviews this PR